app/views/courses/CoursesView.coffee

```diff
@@ -231,3 +238,9 @@ module.exports = class CoursesView extends RootView
     courseInstanceID = $(e.target).data('courseinstance-id')
     window.tracker?.trackEvent 'Students View Levels', category: 'Students', courseID: courseID, courseInstanceID: courseInstanceID, ['Mixpanel']
     application.router.navigate("/students/#{courseID}/#{courseInstanceID}", { trigger: true })
+
+  onClickViewProjectGalleryLink: (e) ->
+    courseID = $(e.target).data('course-id')
+    courseInstanceID = $(e.target).data('courseinstance-id')
+    window.tracker?.trackEvent 'Students View Project Gallery', category: 'Students', courseID: courseID, courseInstanceID: courseInstanceID, ['Mixpanel']
```

Comment:
My understanding is that having different events lets us track which path people take more easily